packages/react-components/react-charts-preview/src/utilities/KeyCodes.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Simulated enum for keycodes. These will get inlined by uglify when used much like an enum
+ *
+ * @public
+ * {@docCategory KeyCodes}
+ */
+export const KeyCodes = {
+  backspace: 8 as 8,
+  tab: 9 as 9,
+  enter: 13 as 13,
+  shift: 16 as 16,
+  ctrl: 17 as 17,
+  alt: 18 as 18,
+  pauseBreak: 19 as 19,
+  capslock: 20 as 20,
+  escape: 27 as 27,
+  space: 32 as 32,
+  pageUp: 33 as 33,
+  pageDown: 34 as 34,
+  end: 35 as 35,
+  home: 36 as 36,
+  left: 37 as 37,
+  up: 38 as 38,
+  right: 39 as 39,
```

Comment:
are we using all of these keys

Reply:
no o we are using only Escape, but kept all of them for reference

Reply:
done